src/modules/displayComments.js

```diff
@@ -0,0 +1,25 @@
+import getComments from './getComments.js';
+
+const countComments = async (data) => {
+  const length = data?.length ?? 0;
+  return length;
+};
+
```

Comment:
well done by implementing the comments functionality , but I wonder why is this function async while it doesn't call any promises you can call it as normal function

Reply:
thank your @souad988 for reviewing our